test/test_sparse_csr.py

```diff
@@ -1208,14 +1208,6 @@ def test_csr_to_block_csr_errors(self, device, dtype):
             with self.assertRaisesRegex(RuntimeError, r"size \(16, 16\) with block size \(5, 5\)"):
                 block_t = t.to_sparse_bsr((5, 5))
 
-    @dtypes(*all_types_and_complex_and(torch.half, torch.bool, torch.bfloat16))
```

Comment:
This test is obsolete. The error cases for `dense->sparse_csr` are now empty batch dims, and when batches do not have the same nnz. Both of these are covered by the new generalized `test_dense_to_from_sparse_compressed`